pro/src/components/layout/form/fields/SiretField/__specs__/SiretField.spec.tsx

```diff
@@ -105,7 +110,7 @@ describe('components | SiretField', () => {
       const humanSiret = '123 456 789 01234'
 
       // when
-      await siretApiValidate(humanSiret, '')
+      await siretApiValidate(humanSiret)
```

Comment:
Tu en à oublié un `l.235`: 
```shell
ERROR in src/components/layout/form/fields/SiretField/__specs__/SiretField.spec.tsx:235:58
TS2554: Expected 1 arguments, but got 2.
    233 |
    234 |       // when
  > 235 |       const errorMessage = await siretApiValidate(siret, '')
        |                                                          ^^
    236 |
    237 |       // then
    238 |       expect(errorMessage).toBe(
```
note: fixé dans mon commit de proposition `TO SQUASH`